### PR TITLE
Create initial version

### PR DESCRIPTION
**Pull request by timothyhinrichs**

### CODE REVIEW

File: roles/k8s_pdp/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  ansible.builtin.fail:
+    msg: "config_mode 'das_discovery' requires das_system_id and das_token and das_tenant"
+  when:
+  - config_mode == 'das_discovery'
+  - das_system_id is not defined or das_token is not defined or das_tenant is not defined
+
+#####################################################################
+# Render manifests into the file system for visibility / debugging
+- name: Log server manifest
+  template:
```
Comment (bcrochet) [inline roles/k8s_pdp/tasks/main.yml]:
```suggestion
  ansible.builtin.template:
```

File: roles/k8s_pdp/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    src: templates/k8s_server.yaml.j2
+    dest: "{{ debug_output_dir }}/server.yaml"
+  when: debug == true
+- name: Log OPA configuration manifest
+  template:
```
Comment (bcrochet) [inline roles/k8s_pdp/tasks/main.yml]:
```suggestion
  ansible.builtin.template:
```

File: roles/k8s_pdp/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+# - name: Create DAS Configuration Manifests
+#   template:
+#     src: templates/k8s_config_das_discovery.yaml.j2
+#     dest: "{{ debug_output_dir }}/das.yaml"
+#   when: debug == true and config_mode == 'das_discovery'
+
+#####################################################################
+# Render and apply those manifests to Kubernetes
+- name: Ensure namespace exists
+  k8s:
```
Comment (bcrochet) [inline roles/k8s_pdp/tasks/main.yml]:
```suggestion
  kubernetes.core.k8s:
```

File: plugins/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Collections Plugins Directory
```
Comment (bcrochet) [inline plugins/README.md]: This can be removed.

**Review by bcrochet — COMMENTED** [review]
Just a few comments about FQCN usage.

File: galaxy.yml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+license:
+- Apache-2.0
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+# license_file: ''
+
+# A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+# requirements as 'namespace' and 'name'
+tags: ["policy", "openpolicyagent", "styra", "authorization", "permission", "declarative", "pdp", "policy_decision_point"]
```
Comment (alisonlhart) [inline galaxy.yml]: For Certification, you'll also need to add one of the following tags to this list:
`application, cloud, database, infrastructure, linux, monitoring, networking, security, storage, tools, windows`
This is for tag-searches.

File: README.md [hunk 6]
```diff
@@ -1 +1,180 @@
...
+
+This ansible collection provides functionality to install OPA as a standlone PDP.  (Often OPA is run as a sidecar as part of a Pod instead of as a standalone service.  Such an installation is out of scope as it requires modifying for example k8s Deployments that make up the application.) There are 3 possible configuration options supported by this ansible collection:
+
+* **barebones**:  No usage of any of the [OPA management configuration](https://www.openpolicyagent.org/docs/latest/management-introduction/) settings.  You are responsible for pushing policies and data into OPA so that it can make decisions.  Not recommended for production use.
+* **bundle**: Includes [bundle configuration](https://www.openpolicyagent.org/docs/latest/management-bundles/) so that policy/data bundles are served from an HTTP server.
+* **das_discovery**: Includes [discovery configuration](https://www.openpolicyagent.org/docs/latest/management-discovery/), which downloads bundle/status/decision-log configuration and then applies it.  Specifically tailored to [Styra DAS](https://www.styra.com/styra-das/).
+
+
```
Comment (alisonlhart) [inline README.md:23]: Add a section on requirements:
- Ansible versions
- Dependencies (if relevant)
- Python versions

File: meta/runtime.yml [hunk 7]
```diff
@@ -0,0 +1,52 @@
+---
+# Collections must specify a minimum required ansible version to upload
+# to galaxy
+# requires_ansible: '>=2.9.10'
```
Comment (alisonlhart) [inline meta/runtime.yml]: The only thing we require in this file is the `requires_ansible` key + value. This value is the minimum required version of ansible. For certification, we require that you support 2+ versions of supported ansible-core with all releases. Refer to this Life Cycle doc for versions of ansible-core that are supported. https://access.redhat.com/support/policy/updates/ansible-automation-platform

You can remove everything in this file that isn't relevant to you. If you do not have any plugin redirects, action groups, python imports, or other modules that need to be defined here (looks like there aren't any modules in the collection) then you can remove everything except the `requires_ansible` statement.

**Review by alisonlhart — COMMENTED** [review]
You'll need to add a changelog file as well. This can be in the root of the collection (CHANGELOG.md, CHANGELOG.rst) or in a changelogs/ directory as a changelog.yml file. 

Tool for generating CHANGELOG.rst files: https://github.com/ansible-community/antsibull-changelog

File: plugins/README.md (hunk 4 above)
Comment (timothyhinrichs) [reply plugins/README.md]: Will do

File: meta/runtime.yml (hunk 7 above)
Comment (timothyhinrichs) [reply meta/runtime.yml]: Great!  Thanks for the help here.

File: roles/k8s_pdp/tasks/main.yml (hunk 1 above)
Comment (timothyhinrichs) [reply roles/k8s_pdp/tasks/main.yml]: Done

File: roles/k8s_pdp/tasks/main.yml (hunk 2 above)
Comment (timothyhinrichs) [reply roles/k8s_pdp/tasks/main.yml]: Done

File: roles/k8s_pdp/tasks/main.yml (hunk 3 above)
Comment (timothyhinrichs) [reply roles/k8s_pdp/tasks/main.yml]: Done

File: plugins/README.md (hunk 4 above)
Comment (timothyhinrichs) [reply plugins/README.md]: Done

File: galaxy.yml (hunk 5 above)
Comment (timothyhinrichs) [reply galaxy.yml]: Done

File: README.md (hunk 6 above)
Comment (timothyhinrichs) [reply README.md:23]: Done

File: meta/runtime.yml (hunk 7 above)
Comment (timothyhinrichs) [reply meta/runtime.yml]: Done

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**